supplier/spring-jdbc-supplier/src/main/java/org/springframework/cloud/fn/supplier/jdbc/JdbcSupplierConfiguration.java

```diff
@@ -71,21 +70,11 @@ public MessageSource<Object> jdbcMessageSource(
 	}
 
 	@Bean(name = "jdbcSupplier")
-	@PollableBean
```

Comment:
Why do we remove PB now?

Reply:
We don't need it any more, since splitter emits only a single `Flux` for the whole stream.
Unlike previously where we converted a `List` for each message into a `Flux`.
Therefore we had to use `@PollableBean` to be able to call `Supplier.get()` when that "short" (cold?) `Flux` for the previous result is exhausted.

Right now we got only a single hot `Flux` as a combination of JDBC source and splitter function.

Reply:
Ah yeh, Flux does not need poll, just initial subscribe then it "flows".